Tasks/ServiceFabricComposeDeploy/task.json

```diff
@@ -15,8 +15,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 0,
-        "Patch": 39
```

Comment:
Semantic versioning, this is a new feature that is backwards compatible so upping the minor version.